sections/jobs.zsh

```diff
@@ -20,14 +21,14 @@ SPACESHIP_JOBS_COLOR="${SPACESHIP_JOBS_COLOR="blue"}"
 spaceship_jobs() {
   [[ $SPACESHIP_JOBS_SHOW == false ]] && return
 
-  local jobs_amount=$( (jobs) | wc -l )
+  local jobs_amount=$( jobs -d | awk '!/pwd/' | wc -l | tr -d " ")
 
   [[ $jobs_amount -gt 0 ]] || return
   [[ $jobs_amount -eq 1 ]] && jobs_amount=''
```

Comment:
I think you'd want to empty the `SPACESHIP_JOBS_SPACE_PREFIX` when jobs amount is `1`, wouldn't you?

Because if not, instead of creating this new variable you could have as well just configured 
`SPACESHIP_JOBS_SYMBOL="✦ "` (with an extra space in the end, which is a common practice in spaceship)

Reply:
Yeah, that makes sense

Reply:
Maybe name it `SPACESHIP_JOBS_AMOUNT_PREFIX` then, to indicate that it has effect only when `amount` is visible? And also maybe add `SPACESHIP_JOBS_AMOUNT_SUFFIX`, just for consistency and completeness 🙂

Reply:
Ok, done!